lib/workers/pr/index.ts

```diff
@@ -56,9 +57,10 @@ function prepareAssigneesReviewers(
 export function prepareLabels(config: RenovateConfig): string[] {
   const labels = config.labels ?? [];
   const addLabels = config.addLabels ?? [];
-  return [...new Set([...labels, ...addLabels])].map((label) =>
-    template.compile(label, config)
-  );
+  return [...new Set([...labels, ...addLabels])]
+    .filter((el) => is.nonEmptyString(el) && !is.emptyStringOrWhitespace(el))
```

Comment:
Can you extract this to a small helper function and use it in both places? Would make the code more readable.

Reply:
added `nonEmptyStringAndNotWhitespace` function